modin/config/envvars.py

```diff
@@ -109,14 +109,11 @@ def _get_default(cls):
                 )
             return "Dask"
         try:
-            from omniscidbe import PyDbEngine  # noqa
-        except ModuleNotFoundError:
-            try:
-                from dbe import PyDbEngine  # noqa
-            except ImportError:
-                pass
-            else:
-                return "Native"
+            from modin.experimental.core.execution.native.implementations.omnisci_on_native.utils import (  # noqa
```

Comment:
As far as I recall you mentioned some LLVM issue about that "option inconsistency" problem - can we probably add some comment here for developers?...

Reply:
Not sure we should add a comment on the LLVM issue so I put a comment on why we import PyDbEngine from this module.